tests/integration/cgroups.bats

```diff
@@ -110,6 +110,9 @@ EOF
 
     runc run -d --console-socket $CONSOLE_SOCKET test_cgroups_permissions
     [ "$status" -eq 0 ]
+    if [ "$CGROUP_UNIFIED" != "no" ]; then
+        check_cgroup_value "cgroup.controllers" "$(cat /sys/fs/cgroup/cgroup.controllers)"
```

Comment:
should be compared against `/sys/fs/cgroup/user.slice/user-${uid}.slice/cgroup.controllers` when rootless && `RUNC_USE_SYSTEMD` is set